Recommendation for ssh and samba add-ons (#11266)
home-assistant/home-assistant.github.io: source/hassio/installation.markdown

6. You will be able to reach your installation at `http://hassio.local:8123` (if your router supports mDNS, otherwise see below).

> 7. It is important to provide yourself proper access and the Hassio custom cli tools. Both the [Samba add-on][samba] and the [SSH add-on][ssh] should be the first add-ons a user should install before making changes to configuration in `/config/` (From the UI choose **Hass.io** which is located in the sidebar and then the Add-on store).

Reviewer: ```suggestion
7. It is important to provide yourself proper access, including the Hass.io CLI tools. Both the [Samba add-on][samba] and the [SSH add-on][ssh] should be the first add-ons you should install, before making changes to the configuration in the `/config/` folder. From the UI choose **Hass.io**, which is located in the sidebar and then the add-on store.
```

Reviewer: The only thing that is missing IMHO, is WHY one needs this.
However, still a major improvement, so 👍
